### PR TITLE
Feature/prl 4259 changes to generic screen 

### PR DESCRIPTION
**Pull request by JamesCutts00**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/private-law/json/State.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "Name": "Draft",
     "Description": "Draft",
     "DisplayOrder": 1,
-    "TitleDisplay": "# #${[CASE_REFERENCE]}\n ## Case Name: ${applicantCaseName}\n ## Family Man ID: ${familymanCaseNumber}"
+    "TitleDisplay": "## ${applicantCaseName}\n ## Familyman ID: ${familymanCaseNumber}\n ## Casenumber: ${[CASE_REFERENCE]}"
```
Comment (shashisk97) [inline definitions/private-law/json/State.json:9]: Will this change make case number to be displayed twice in the page?

**Review by shashisk97 — COMMENTED** [review]
Field display order needs to be checked one more time to make sure the sequence is correct.

File: definitions/private-law/json/CaseEventToFields/DraftAnOrder/Page16 -Hearing Data/CaseEventToFields.json [hunk 2]
```diff
@@ -3,14 +3,36 @@
...
     "CaseEventID": "draftAnOrder",
-    "CaseFieldID": "hasJudgeProvidedHearingDetails",
-    "DisplayContext": "MANDATORY",
+    "CaseFieldID": "selectedOrderHearingLabel",
+    "DisplayContext": "READONLY",
     "PageID": 16,
     "PageDisplayOrder": 16,
     "PageFieldDisplayOrder": 1,
     "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/generate-doc",
+    "FieldShowCondition": "createSelectOrderOptions!=\"blankOrderOrDirections\"",
```
[the remark this answers is not in the document]
Comment (JamesCutts00) [reply definitions/private-law/json/CaseEventToFields/DraftAnOrder/Page16 -Hearing Data/CaseEventToFields.json:12]: if the order is blankorderordirections, the label will be set as the c21 specific order ie blank order: other etc else the label will be the general label ie Parental responsibility order

File: definitions/private-law/json/CaseEventToFields/DraftAnOrder/Page4/CaseEventToFields.json [hunk 3]
```diff
@@ -9,6 +9,19 @@
     "PageID": 4,
     "PageDisplayOrder": 4,
     "ShowSummaryChangeOption": "Y",
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/selected-order",
```
[the remark this answers is not in the document]
Comment (JamesCutts00) [reply definitions/private-law/json/CaseEventToFields/DraftAnOrder/Page4/CaseEventToFields.json:12]: deleated the call back, reasoning for two labels explained in previous comment

File: definitions/private-law/json/CaseEventToFields/DraftOrderAdmin/Page14-Hearing Data-solicitor/CaseEventToFields.json [hunk 4]
```diff
@@ -1,4 +1,16 @@
 [
+  {
+    "LiveFrom": "31/03/2022",
+    "CaseTypeID": "PRLAPPS",
+    "CaseEventID": "adminEditAndApproveAnOrder",
+    "CaseFieldID": "orderNameSolicitorLabel",
+    "PageFieldDisplayOrder": 2,
```
[the remark this answers is not in the document]
Comment (JamesCutts00) [reply definitions/private-law/json/CaseEventToFields/DraftOrderAdmin/Page14-Hearing Data-solicitor/CaseEventToFields.json:7]: set to 1

File: definitions/private-law/json/CaseEventToFields/DraftOrderAdmin/Page15-HearingData/CaseEventToFields.json [hunk 5]
```diff
@@ -3,12 +3,22 @@
...
+  },
+  {
+    "LiveFrom": "22/10/2022",
+    "CaseTypeID": "PRLAPPS",
+    "CaseEventID": "adminEditAndApproveAnOrder",
+    "CaseFieldID": "ordersHearingDetails",
+    "DisplayContext": "COMPLEX",
+    "PageID": 15,
+    "PageDisplayOrder": 15,
+    "PageFieldDisplayOrder": 3
```
[the remark this answers is not in the document]
Comment (JamesCutts00) [reply definitions/private-law/json/CaseEventToFields/DraftOrderAdmin/Page15-HearingData/CaseEventToFields.json:22]: set to 2

File: definitions/private-law/json/CaseEventToFields/DraftOrderAdmin/Page5/CaseEventToFields.json [hunk 6]
```diff
@@ -10,7 +10,19 @@
     "PageDisplayOrder": 5,
     "ShowSummaryChangeOption": "N",
     "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/judge-or-admin-populate-draft-order-custom-fields",
-    "PageShowCondition": "doYouWantToEditTheOrder=\"Yes\" AND orderType!=\"noticeOfProceedingsParties\" AND orderType!=\"noticeOfProceedingsNonParties\""
+    "PageShowCondition": "doYouWantToEditTheOrder=\"Yes\" AND orderType!=\"noticeOfProceedingsParties\" AND orderType!=\"noticeOfProceedingsNonParties\"",
```
[the remark this answers is not in the document]
Comment (JamesCutts00) [reply definitions/private-law/json/CaseEventToFields/DraftOrderAdmin/Page5/CaseEventToFields.json:13]: Yes, have tried for a c6 form, made changes as neccessary

File: definitions/private-law/json/CaseEventToFields/ManageOrders/Page 20 - Common (Preview Order)/CaseEventToFields.json [hunk 7]
```diff
@@ -9,31 +9,32 @@
...
     "CaseEventID": "manageOrders",
-    "CaseFieldID": "checkYourOrderLabel",
+    "CaseFieldID": "checkYourOrder",
     "PageFieldDisplayOrder": 3,
     "DisplayContext": "READONLY",
     "PageID": 20,
     "PageDisplayOrder": 20,
-    "ShowSummaryChangeOption": "Y"
+    "ShowSummaryChangeOption": "Y",
+    "PageShowCondition": "manageOrdersOptions=\"createAnOrder\""
```
[the remark this answers is not in the document]
Comment (JamesCutts00) [reply definitions/private-law/json/CaseEventToFields/ManageOrders/Page 20 - Common (Preview Order)/CaseEventToFields.json:37]: changed so it is the first field in the page

File: definitions/private-law/json/CaseEventToFields/ManageOrders/Page 5 - Common/CaseEventToFields.json [hunk 8]
```diff
@@ -9,27 +9,51 @@
...
+  {
+    "LiveFrom": "31/03/2022",
+    "CaseTypeID": "PRLAPPS",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "selectedOrderLabel",
+    "PageFieldDisplayOrder": 4,
+    "DisplayContext": "READONLY",
+    "PageID": 5,
+    "PageDisplayOrder": 5,
+    "ShowSummaryChangeOption": "N",
```
[the remark this answers is not in the document]
Comment (JamesCutts00) [reply definitions/private-law/json/CaseEventToFields/ManageOrders/Page 5 - Common/CaseEventToFields.json:48]: set to y

File: definitions/private-law/json/State.json (hunk 1 above)
Comment (JamesCutts00) [reply definitions/private-law/json/State.json:9]: no it has not

**Pull request closed without merging**